Update roles-for-resource-and-storage.md to prevent 403 error (#124188)
Azure/azure-content: articles/ai-services/language-service/custom-named-entity-recognition/includes/roles-for-resource-and-storage.md

5. Select **Select members**
6. Select your subscription, and **Language** as the managed identity. You can search for user names in the **Select** field. 

### Roles for your user
> [!IMPORTANT]

> > If you skip this step you will have a 403 error when trying to connect to your custom project, it is important that your current user has this role to access storage account blob data even if you are the owner of the storage account.

Reviewer: ```suggestion
> If you skip this step, you'll have a 403 error when trying to connect to your custom project. It's important that your current user has this role to access storage account blob data, even if you're the owner of the storage account.
```
